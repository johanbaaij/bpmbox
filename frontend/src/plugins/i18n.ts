import Vue from "vue";
import VueI18n, { LocaleMessages } from "vue-i18n";
import localesEn from "@/locales/en.json";

Vue.use(VueI18n);

const en: LocaleMessages = { en: localesEn };

export default new VueI18n({
  locale: process.env.VUE_APP_I18N_LOCALE || "en",
  fallbackLocale: process.env.VUE_APP_I18N_FALLBACK_LOCALE || "en",
  messages: en
});
