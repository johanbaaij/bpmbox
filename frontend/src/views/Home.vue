<template>
  <v-container class="fill-height" fluid>
    <v-row>
      <v-col> </v-col>
    </v-row>
  </v-container>
</template>

<script>
export default {
  name: "home"
};
</script>
