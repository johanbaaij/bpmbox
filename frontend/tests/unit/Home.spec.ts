import "./setup";

// Dependencies
import Vuetify from "vuetify";
import i18n from "@/plugins/i18n";

// Components
import Home from "../../src/views/Home.vue";

// Utilities
import { mount, createLocalVue } from "@vue/test-utils";

const localVue = createLocalVue();

describe("Home.vue", () => {
  let vuetify: any;

  beforeEach(() => {
    vuetify = new Vuetify();
  });

  it("should match snapshot", () => {
    const wrapper = mount(Home, {
      localVue,
      vuetify,
      i18n
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
