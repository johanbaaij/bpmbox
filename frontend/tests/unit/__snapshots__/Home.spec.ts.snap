// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue should match snapshot 1`] = `
<div class="container container--fluid">
  <div class="row">
    <div class="col">
      <p>Enter a release id.</p>
    </div>
  </div>
  <div class="row fill-height">
    <div class="col"><span name="model" mode="aggressive"><span><div class="v-input theme--light v-text-field v-text-field--single-line v-text-field--solo v-text-field--enclosed v-text-field--placeholder"><div class="v-input__control"><div class="v-input__slot"><div class="v-input__prepend-inner"><div class="v-input__icon v-input__icon--prepend-inner"><i aria-hidden="true" class="v-icon notranslate mdi mdi-magnify theme--light"></i></div></div><div class="v-text-field__slot"><input id="input-8" placeholder="5267551" type="text"></div><div class="v-input__append-inner"><button type="button" class="v-btn v-btn--contained theme--light v-size--default"><span class="v-btn__content">Go</span></button></div>
  </div>
  <div class="v-text-field__details">
    <div class="v-messages theme--light">
      <div class="v-messages__wrapper"></div>
    </div>
  </div>
</div>
</div></span>
<p>

</p></span></div>
</div>
</div>
`;
